solutions/usaco-1060.mdx

```diff
@@ -2,7 +2,7 @@
 id: usaco-1060
 source: USACO Bronze 2020 December
 title: Daisy Chains
-author: Danh Ta Chi Thanh
+author: Sathvik Chundru [Java], Danh Ta Chi Thanh [CPP]
```

Comment:
remove the languages

Reply:
k

Reply:
done can u approve or make another suggestion